target/linux/mediatek/dts/mt7622-netgear-wax206.dts

```diff
@@ -0,0 +1,603 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2022, Marcel Ziswiler <marcel@ziswiler.com> */
+
+/dts-v1/;
+#include "mt7622.dtsi"
+#include "mt6380.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Netgear WAX206";
+	compatible = "netgear,wax206", "mediatek,mt7622";
+
+	aliases {
+		ethernet0 = &gmac0;
+//		label-mac-device = &gmac0;
+//		label-mac-device = &wan;
```

Comment:
Please remove these commented sections if they are not needed

Reply:
There are also some other similar comments in this file

Reply:
@hauke I are right, there are still some comments I haven't removed. I'll remove the unnecessary comment.

Do I do I force push again (with maybe a rebase) or push some commit for requested changes and then do a force push when reviewers are happy?

Reply:
Remove the not needed comments, which is most.